data-processing-lib/ray/src/data_processing_ray/runtime/ray/transform_orchestrator.py

```diff
@@ -142,7 +142,8 @@ def orchestrate(
         # Compute execution statistics
         logger.debug("Computing execution stats")
         stats = runtime.compute_execution_stats(ray.get(statistics.get_execution_stats.remote()))
-        stats["processing_time"] = round(stats["processing_time"], 3)
+        if "processing_time" in stats:
```

Comment:
I think, value 0 for processing_time can be misleading.

Reply:
The only way `stats["processing_time"] = round(stats["processing_time"], 3)` can throw an exception is in the case where either:
1. init method in file processor fails due to inability to create transform
2. All of the transform invocations failed and "processing time" was never reported by file processor

In both cases, there is really no statistics of interest to be collected. 
But if you really want to avoid this exception the right thing to do (in my opinion) is to do what Shivdeep is doing:

```
      if "processing_time" in stats:
            stats["processing_time"] = round(stats["processing_time"], 3)
```
This will return metadata with no processing time value, which is correct in this case

Note that if you want to go for this fix, please fix it in all 3 orchestrators - Python, Ray and Spark. Not just Ray